File: JavaScript/Easy/Q1431-KidsWithTheGreatestNumberOfCandies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function (candies, extraCandies) {
    const greatest = [];

    for (let i = 0; i < candies.length; i++) {
        const candy = candies[i] + extraCandies;
        const leftCandies = [...candies];
        leftCandies.splice(i, 1);
        if (candy >= Math.max(...leftCandies)) greatest.push(true);
        else greatest.push(false);
    }

    return greatest;
};

File: Python/Leetcode75/Easy/Array/Q1431-KidsWithTheGreatestNumberOfCandies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        g_candies = []

        for i in range(len(candies)):
            if candies[i] + extraCandies >= max(candies):
                g_candies.append(True)
            else:
                g_candies.append(False)

        return g_candies